s27816-MichaelWajda/zajecia1/instalacja_debian.txt

```diff
@@ -0,0 +1,454 @@
+danej partycji, co pozwala na przechowywanie dodatkowych metadanych plików.
+Reserved blocks: ilość bloków na każdej partycji zarezerwowanych dla 
+root'a, zostawiamy bez zmian.
+Configure ISCSI volumes -  konfiguracja dysku, do którego nie mamy fizycznie, 
+ale do którego możemy uzyskać dostęp przez sieć (np. serwer firmy).
+29) Po dokonaniu partycjonowana wybieramy 'Finish partitioning and write 
+changes to disk' a następnie 'Yes' aby rozpocząć instalację podstawy 
+systemu (z nośnika). 
+Nie chcemy wybierać kolejnej płyty, więc zapytani wybieramy 'No', aby 
+rozpocząć ściąganie reszty systemu z sieci.
+Wybieramy terytorium na którym się znajdujemy 'Poland' a następnie 
+ftp.task.gda.pl, znajdującego się na politechnice Gdańskiej, czyli 
+najbliżej w tym przypadku.
+Proxy nie używamy, zostawiamy więc pusta przestrzeń i przechodzimy dalej.
+Na zapytanie o wzięcie udziału w ankiecie popularności pakietów wybieramy 
+'No', gdyż jest to instalacja w celu przeprowadzenia jedynie w celu 
+przeprowadzenia ćwiczenia.
+30) Software Selection
+Tu zaznaczamy i odznaczamy spacją. Enter przejdzie dalej w instalacji.
+Debian desktop environment - Środowiska graficzne pozostawiamy wyłączone ze 
+względu na założenia instalacji
+GNOME - jedne z najbardziej popularnych środowisk pulpitu w systemach Linux. 
+zaawansowane funkcje, dużą ilość rozszerzeń i wtyczek. Potencjalnie 
+niestabilny.
+Xfce - Jedno z najmniejszych i najszybszych środowisk graficznych
+GNOME Flashback - starsza wersja GNOME, w okolicach GNOME 2 wykorzystuje 
+więcej zasobów niż Xfce.
+KDE Plasma - kolejne popularne środowisko pulpitu, które oferuje zaawansowane 
+funkcje i dużą elastyczność konfiguracji. Posiada nowoczesny i estetyczny 
+interfejs użytkownika konsumujący dużą ilość zasobów.
+Cinnamon - Środowisko graficzne zbliżone do klasycznego interface'u Wndowsa.
+MATE - kontynuacja GNOME 2, które zachowuje tradycyjny wygląd i funkcje GNOME 
+2, ale z nowszą architekturą i aktualizacjami. 
+LXDE - Minimalistyczne środowisko pulpitu, które zapewnia prosty i oszczędny 
+interfejs użytkownika, przy minimalnym zużyciu zasobów systemowych.
+LXQt - lekkie środowisko pulpitu, które jest oparte na bibliotekach Qt. 
+Zapewnia ono prosty interfejs użytkownika oraz niskie zużycie zasobów 
+systemowych.
+Web Server - Opcja instalująca serwer WWW
+SSH Server - (Secure Shell) umożliwia zdalne logowanie się do systemu lub 
+serwera oraz wykonywanie poleceń przez Internet w sposób bezpieczny.
+Standard system utilities - podstawowe narzędzia (jak np. grep)
+Wybieramy jedynie SSH Server  i Standard system utilities ponieważ nie 
+potrzebujemy środowisk graficznych, przechodzimy dalej wybierając 'Continue'.
+31) GRUB Boot Loader to program, który pozwala na wybór systemu operacyjnego 
+do uruchomienia i innych opcji podczas startu komputera. Jako, że jest to 
+jedyny boot loader na dysku (jako że instalujemy system bez EFI), 
+to instalujemy go w MBR (Master Boot Record). Wybieramy więc 'Yes' a 
```

Comment:
Gdyby było więcej loaderów to gdzie powinien zostać taki bootloader zainstalowany wtedy?